### ISSUE TITLE
Missing Test Case - 223. Rectangle Area

### ISSUE THREAD

**Issue by sunil1906** [LeetCode-Feedback/LeetCode-Feedback#7903]
#### Your LeetCode username
sunil1906

#### Category of the bug
Solution

#### Description of the bug
Question description says "The first rectangle is defined by its bottom-left corner (ax1, ay1) and its top-right corner (ax2, ay2)". 

Therfore, ax2 should be greater than or equal to ax1 and ay2 should be greater than or equal to  ay1. But problems setters solution code is accepting the test case for ax2 < ax1 also which should not accept. 

#### Code you used for Submit/Run operation

```
// 223. Rectangle Area
class Solution {
public:
    int computeArea(int ax1, int ay1, int ax2, int ay2, int bx1, int by1, int bx2, int by2) {
        int ans = ( (ax2 - ax1) * (ay2 - ay1) ) + ( (bx2 - bx1) * (by2 - by1) );
        int x_overlap = max( ((ax2 - ax1) + (bx2 - bx1)) - (max(ax2, bx2) - min(ax1, bx1)) , 0);
                        // Sum of width separately    Sum of Width Overlapped
        int y_overlap = max( ((ay2 - ay1) + (by2 - by1)) - (max(ay2, by2) - min(ay1, by1)) , 0);
                        // Sum of Height separately    Sum of Height Overlapped
        ans -= x_overlap * y_overlap;
        return ans;
    }
};
```

#### Language used for code
C++

#### Expected behavior
Expected behaviour: Test case:

1
0
0
1

10
10
11
11

should give output as invalid as per description. But it gives an output.

#### Screenshots
![image](https://user-images.githubusercontent.com/53760997/175774288-c565d88d-3fdd-4955-8712-dc9ebf4f67fa.png)
How can accepted solution fail a custom test case? Beacause there is a check missing in problem setters solution to check left bottom point should be lesser than upper right value.

In short, you should include this in your testing solution
if(ax2 < ax1 or ay2 < ay1 or bx2 < bx1 or by2 < by1)  ---> Give Invalid Test case.

**Comment by LC-Pam** [comment]
Thank you for reaching out to us. I've relayed this issue to our team to investigate.

**Comment by LC-Pam** [comment]
Thank you for your time. We've used your feedback to update the problem. Your LeetCode account has received 100 LeetCoins as the reward for this feedback. We appreciate your support!
